libs/src/services/solanaWeb3Manager/transactionHandler.ts

```diff
@@ -159,10 +202,9 @@ class TransactionHandler {
     // Send the txn
 
     const sendRawTransaction = async () => {
-      return this.connection.sendRawTransaction(rawTransaction, {
+      return await this.connection.sendRawTransaction(rawTransaction, {
         skipPreflight:
           skipPreflight === null ? this.skipPreflight : skipPreflight,
```

Comment:
Is `commitment` not a arg on `sendRawTransaction`? This is surprising to me

Reply:
https://solana-labs.github.io/solana-web3.js/modules.html#SendOptions